ccloud/docs/beginner-cloud.rst

```diff
@@ -637,73 +585,40 @@ Run kafka-connect-datagen connector with wildcard ACLs
 
    .. code-block:: text
 
-         DATA=$( cat << EOF
-         {
-            "name": "datagen-demo-topic-3",
-            "config": {
-              "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
-              "kafka.topic": "demo-topic-3",
-              "quickstart": "pageviews",
-              "key.converter": "org.apache.kafka.connect.storage.StringConverter",
-              "value.converter": "org.apache.kafka.connect.json.JsonConverter",
-              "value.converter.schemas.enable": "false",
-              "max.interval": 5000,
-              "iterations": 1000,
-              "tasks.max": "1"
-            }
-         }
-         EOF
-         )
-
-         curl --silent --output /dev/null -X POST -H "Content-Type: application/json" --data "${DATA}" http://localhost:8083/connectors
-
-
-#. Wait about 20 seconds for the kafka-connect-datagen connector to start producing messages.
+      Creating connector from datagen_ccloud_pageviews.json
+
+      2020-12-16T16:10:09.732-0700 [DEBUG] ConnectService.Create request: POST https://confluent.cloud/api/accounts/env-5qz2q/clusters/lkc-x6m01/connectors Body:{"name":"datagen_ccloud_pageviews","config":{"connector.class":"DatagenSource","iterations":"1000000000","kafka.api.key":"ESN5FSNDHOFFSUEV","kafka.api.secret":"nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB","kafka.topic":"demo-topic-3","max.interval":"500","name":"datagen_ccloud_pageviews","output.data.format":"JSON","quickstart":"PAGEVIEWS","tasks.max":"1"}}
+      2020-12-16T16:10:21.658-0700 [DEBUG] ConnectService.Create response: 201 Created Body: {"name":"datagen_ccloud_pageviews","type":"source","config":{"cloud.environment":"prod","cloud.provider":"aws","connector.class":"DatagenSource","iterations":"1000000000","kafka.api.key":"****************","kafka.api.secret":"****************","kafka.dedicated":"false","kafka.endpoint":"SASL_SSL://pkc-4kgmg.us-west-2.aws.confluent.cloud:9092","kafka.region":"us-west-2","kafka.topic":"demo-topic-3","kafka.user.id":"152738","max.interval":"500","name":"datagen_ccloud_pageviews","output.data.format":"JSON","quickstart":"PAGEVIEWS","tasks.max":"1"},"tasks":[]} request: POST https://confluent.cloud/api/accounts/env-5qz2q/clusters/lkc-x6m01/connectors Body:{"name":"datagen_ccloud_pageviews","config":{"connector.class":"DatagenSource","iterations":"1000000000","kafka.api.key":"ESN5FSNDHOFFSUEV","kafka.api.secret":"nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB","kafka.topic":"demo-topic-3","max.interval":"500","name":"datagen_ccloud_pageviews","output.data.format":"JSON","quickstart":"PAGEVIEWS","tasks.max":"1"}}
+      Created connector datagen_ccloud_pageviews lcc-zno83
 
 #. Run the following command to verify connector is running:
 
    .. code-block:: bash
 
-      curl --silent http://localhost:8083/connectors/datagen-demo-topic-3/status | jq -r '.'
+      ccloud::wait_for_connector_up datagen_ccloud_pageviews.json 300
```

Comment:
This is a library function which is great for automation (and the `start.sh` script) but for the docs it is more instructive to have the user learn the Confluent Cloud CLI command `ccloud connector <list>` (or `<describe>`) instead.